cloud/aws/aws_file_system.h

```diff
@@ -42,6 +42,10 @@ namespace ROCKSDB_NAMESPACE {
 // AwsFileSystem internally share PosixFileSystem for sharing common file system
 // resources.
 //
+// NOTE: The AWS SDK must be initialized before any AwsFileSystem is
+// constructed, and remain active (Aws::ShutdownAPI() not called) as long as any
+// AwsFileSystem objects exist.
```

Comment:
This comment belongs in the public header, here: https://github.com/rockset/rocksdb-cloud/blob/513b855d17bd2f58831c5019a5e9e938824e1fae/include/rocksdb/cloud/cloud_file_system.h#L661-L671